src/herbpy/herbrobot.py

```diff
@@ -271,7 +270,93 @@ def SetStiffness(self, stiffness):
         self.head.SetStiffness(stiffness)
         self.left_arm.SetStiffness(stiffness)
         self.right_arm.SetStiffness(stiffness)
-
+        
+    def DetectHuman(self, env, orhuman='kin2_or', load_hum=True,                           
+                    enable_legs=True, segway_sim=True, 
+                    base_frame='/map' , kin_frame='/head/skel_depth_frame2',
+                    hrc=False, herb_sim=True, 
+                    continuous=True, action='stamp'):
```

Comment:
I don't think this should be a function HerbPy. It enters an infinite `while not rospy.is_shutdown()` loop and will never return in normal operation. I suggest moving this to a separate script file, much like `console.py`, that we can run from the command line for visualization purposes.